libvoikko/typescript/libvoikko.d.ts

```diff
@@ -0,0 +1,198 @@
+export = Libvoikko;
+
+declare class Libvoikko {
+  constructor();
+  init(language: string): Voikko;
```

Comment:
Missing optional parameter here
```suggestion
  init(language: string, path?: string): Voikko;
```